src/demos/head.js

```diff
@@ -25,4 +25,6 @@
   loadCss("build/calcite-app.css");
   loadScript("build/calcite-app.esm.js", { type: "module" });
   loadScript("build/calcite-app.js", { noModule: true });
+  loadScript("node_modules/@esri/calcite-components/dist/calcite/calcite.esm.js", { type: "module" });
```

Comment:
Thinking about the order of load here. Do you foresee any situations where there's something both calcite and calcite-apps are trying to set? If so do you know which one wins priority, the first or the second to do so? And in that case would we want ours or theirs to take priority?